app/javascript/components/organization/registration/Step2.jsx

```diff
@@ -17,7 +17,7 @@ class Step2 extends React.Component {
       name: PropTypes.string,
       path: PropTypes.string,
       preview: PropTypes.string
-    }).isRequired,
+    }),
```

Comment:
@ethanlee16 why did we remove the `isRequired`?

Reply:
It's a nullable now!